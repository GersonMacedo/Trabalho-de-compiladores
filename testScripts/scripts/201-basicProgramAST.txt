! -s TREE
! should print correctly the AST for the basic program
program p ;
begin
end .