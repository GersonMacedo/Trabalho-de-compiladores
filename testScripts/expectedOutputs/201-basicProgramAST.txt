
Performing up to syntactic analysis and printing the AST tree at testScripts/scripts/201-basicProgramAST.txt
No errors found

Programa 'p'
Declarações:
Corpo:
